### DISCUSSION TITLE
how to update roblox version?

### DISCUSSION THREAD

**Discussion by mamalubitlal** [vinegarhq/sober#418] [Q&A]
how to update roblox version to more recent one?i tried uninstalling sober and installing but it saves my data

**Comment by k1yrix** [comment]
Sober uses a fixed version modal. You can't do that until they update to a newer fixed version.

**Reply by mamalubitlal** [reply]
> Sober uses a fixed version modal. You can't do that until they update to a newer fixed version.

can  i install all sober data an runtime then?
